src/components/Modal/index.js

```diff
@@ -12,37 +12,25 @@ const Wrapper = ({children}) =>
   wrapWithClass(style.inner, children)
 
 class Modal extends Component {
-  constructor (props) {
-    super(props)
-    this.state = {isOpen: false}
-  }
-
-  openModal = () => {
-    this.setState({isOpen: true})
-  }
-
-  onRequestClose = () => {
-    this.setState({isOpen: false})
-  }
-
   render () {
     const { translate, isFullScreen } = this.props
     return (
       <ReactModal
-        isOpen={this.state.isOpen || this.props.isOpen}
-        onRequestClose={this.props.onRequestClose || this.onRequestClose}
+        isOpen={this.props.isOpen}
+        onRequestClose={this.props.onRequestClose}
         portalClassName={style.portal}
         overlayClassName={style.overlay}
         bodyClassName={style.modalBody}
         className={style.inner}
         shouldCloseOnOverlayClick={true}
         closeTimeoutMS={MODAL_ANIMATION_DURATION}
+        ariaHideApp={false}
```

Comment:
what does this do?

Reply:
It prevents `react-modal` from throwing `You must set an element with Modal.setAppElement(el) to make this accessible`. Thinking about it, this is probably not the best way of fixing the error because this will make the content behind the modal readable by screen readers. There is another fix and it involves passing `appElement={document.body}` or  `appElement={document.getElementById('onfido-mount')}`. I think this is a better solution because it doesn't break screen readers. See discussion here https://github.com/reactjs/react-modal/issues/576